common-util/Login/config.jsx

```diff
@@ -1,15 +1,16 @@
-import { cookieStorage, createStorage } from 'wagmi';
-import { mainnet, goerli } from 'wagmi/chains';
+import { cookieStorage, createStorage, http } from 'wagmi';
+import { mainnet, goerli, base} from 'wagmi/chains';
 import { defaultWagmiConfig } from '@web3modal/wagmi';
 
 import { SITE_DESCRIPTION, SITE_TITLE, SITE_URL } from 'util/constants';
+import { RPC_URLS } from 'common-util/Contracts';
 
-// if the PFP_URL contains staging, use goerli, else use mainnet
+// if the PFP_URL contains staging, include goerli
 export const SUPPORTED_CHAINS = (
   process.env.NEXT_PUBLIC_PFP_URL || ''
 ).includes('staging')
-  ? [goerli, mainnet]
-  : [mainnet];
+  ? [goerli, mainnet, base]
```

Comment:
goerli is dead now
```suggestion
  ? [mainnet, base]
```

Reply:
yes yes I know... but there are many places where it should be removed in contribute, I suggest to figure out a general approach for all apps and do it later